15314f1ddb882700dc9fabc5ca961943/update/sys_script_56d4558c97600a10b7daf896f053af78.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?><record_update sys_domain="global" table="sys_script">
+    <sys_script action="INSERT_OR_UPDATE">
+        <abort_action>false</abort_action>
+        <access>package_private</access>
+        <action_delete>false</action_delete>
+        <action_insert>true</action_insert>
+        <action_query>false</action_query>
+        <action_update>false</action_update>
+        <active>true</active>
+        <add_message>false</add_message>
+        <advanced>true</advanced>
+        <change_fields>false</change_fields>
+        <client_callable>false</client_callable>
+        <collection>x_illu2_illumio_illumio_label_dimension</collection>
+        <condition/>
+        <description/>
+        <execute_function>false</execute_function>
+        <filter_condition/>
+        <is_rest>false</is_rest>
+        <message/>
+        <name>Create Label Dimension</name>
+        <order>100</order>
+        <priority>100</priority>
+        <rest_method/>
+        <rest_method_text/>
+        <rest_service/>
+        <rest_service_text/>
+        <rest_variables/>
+        <role_conditions/>
+        <script><![CDATA[(function executeRule(current, previous /*null when async*/ ) {
+    var key = current.getValue("key").trim();
+    var configSysId = current.getValue("illumio_pce_configuration").trim();
+
+    // Get all the values and push to Illumio PCE. If error from Illumio, we abort the action.
+    var details = {
+        "key": key,
+        "display_name_singular": current.getValue("display_name_singular").trim(),
+        "display_name_plural": current.getValue("display_name_plural").trim(),
+        "label_type_initial": current.getValue("label_type_initial").trim(),
+        "illumio_pce_configuration": configSysId
+    };
+
+    var IlUtils = new IllumioUtils();
+    var matchFound = IlUtils.validateLabelDimensionCreation(key, configSysId);
+    if (!matchFound) {
```

Comment:
what happens if a match is found? is the user notified of it?

Reply:
Yes, in case a label dimension already exists on Illumio with the same key. A message is registered in the logged and the label dimension record is updated as per values obtained from Illumio.